### DISCUSSION TITLE
(resolved)Configuration is not consistent with adapter state/backup! PAN ID

### DISCUSSION THREAD

**Discussion by pimboli** [Koenkk/zigbee2mqtt#9394] [Devices]
Good morning,

just realized that my Zigbee isn't working anymore.
I searched for solutions and read that I should change the PAN ID to 6755, but without success.
Also different in my case is that the adapter is set to adapter=0
My adapter is a CC2531.

Any idea? Thanks

`[08:49:40] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.21.2 start
> node index.js
Zigbee2MQTT:error 2021-10-29 08:49:47: Configuration is not consistent with adapter state/backup!
Zigbee2MQTT:error 2021-10-29 08:49:47: - PAN ID: configured=6754, adapter=0
Zigbee2MQTT:error 2021-10-29 08:49:47: - Extended PAN ID: configured=00124b001cde54f6, adapter=00124b001cde54f6
Zigbee2MQTT:error 2021-10-29 08:49:47: - Network Key: configured=01030507090b0d0f00020406080a0c0d, adapter=01030507090b0d0f00020406080a0c0d
Zigbee2MQTT:error 2021-10-29 08:49:47: - Channel List: configured=11, adapter=11
Zigbee2MQTT:error 2021-10-29 08:49:47: Please update configuration to prevent further issues.
Zigbee2MQTT:error 2021-10-29 08:49:47: If you wish to re-commission your network, please remove coordinator backup at /config/zigbee2mqtt/coordinator_backup.json.
Zigbee2MQTT:error 2021-10-29 08:49:47: Re-commissioning your network will require re-pairing of all devices!
Zigbee2MQTT:error 2021-10-29 08:49:47: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-10-29 08:49:47: Failed to start zigbee
Zigbee2MQTT:error 2021-10-29 08:49:47: Check https://www.zigbee2mqtt.io/information/FAQ.html#help-zigbee2mqtt-fails-to-start for possible solutions
Zigbee2MQTT:error 2021-10-29 08:49:47: Exiting...
Zigbee2MQTT:error 2021-10-29 08:49:47: Error: startup failed - configuration-adapter mismatch - see logs above for more information`

**Comment by Koenkk** [comment]
Looks like your cc2531 crashes, try reflashing it.

**Comment by lux73** [comment]
and consider to Upgrade the old rubbish CC2531 to at leaset CC2652xx Stick!

i can really recommend to change it - no Problems anymore after change... ;-)

**Reply by Koenkk** [reply]
That is an even better solution indeed

**Comment by pimboli** [comment]
Hello,

thank you for your feedback, I don't know why but installing a 3 days old backup solved the problem, even if software versions were the same.

I ordered a new dongle, can I simply replace it or do I need to reset and bind all actors again?

**Reply by psi-4ward** [reply]
https://www.zigbee2mqtt.io/guide/faq/#what-does-and-does-not-require-repairing-of-all-devices

**Reply by pimboli** [reply]
Thank you, successfully replaced it.
